src/components/PanelCard/PanelCard.styles.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import Image from "next/image";
+import styled from "styled-components";
+import { Icon } from "../Icon/Icon";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  max-width: 20rem;
+  min-height: 30rem;
+`;
+export const Description = styled.p`
+  color: white;
+  font-size: 1rem;
+  text-align: center;
+  padding: 1rem;
+`;
+
+export const DescriptionContainer = styled.div`
+  position: absolute;
+  overflow-y: scroll;
+  background-color: #00000090;
+  width: 100%;
+  height: 100%;
+  border-radius: 0.5rem;
+  top: 0;
+  display: none;
+  animation: fadeIn 300ms linear forwards;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const ThumbContainer = styled.div`
+  position: relative;
+  aspect-ratio: 16 / 19;
+  width: 100%;
+  height: 100%;
+
+  &:hover ${DescriptionContainer} {
+    display: flex;
+  }
+`;
+
+export const Link = styled.a`
+  overflow: hidden;
+  width: 100%;
+  height: auto;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+`;
```

Comment:
Add this to the top of the page:
```js
import DefaultLink from "next/link";
```
And this instead of this Link
```suggestion
export const Link = styled(DefaultLink)`
  overflow: hidden;
  width: 100%;
  height: auto;
  margin: 0;
  padding: 0;
  text-decoration: none;
  display: flex;
  flex-flow: column;
  gap: 1rem;
`;
```